protocol.md

```diff
@@ -190,33 +190,39 @@ Cryptographic algoritms used by PFS are described in [Perfect forward secrecy se
 ## Topic
 
 There are two types of Whisper topics the protocol uses:
-* static topic for `user-message` message type (also called _contact discovery topic_)
-* dynamic topic based on a chat name for `public-group-user-message` message type.
+* topics for private chats (also called _partitioned topics_)
+* dynamic topic based on a chat name for public chats
 
-The static topic is always the same and its hex representation is `0xf8946aac`. In fact, _the contact discovery topic_ is calculated using a dynamic topic algorithm described below with a constant name `contact-discovery`.
+Having partitoned private chat topics reduce darkness, but they allow keeping traffic under control.
 
-Having only one topic for all private chats has an advantage as it's very hard to figure out who talks to who. A drawback is that everyone receives everyone's messages but they can decrypt only these they have private keys for.
+Each user has a unique-ish topic he listens to for 1-1 messages. Number of these topics are capped to N=5000.
+That gives a decent balance between darkness
```

Comment:
Imprecise statement, perhaps something on receiver anonymity being proportional and anonymity set decreasing?